string[] file_paths
---
bool success
string message
string[] export_path
